### DISCUSSION TITLE
idk how to use NUTEditor

### DISCUSSION THREAD

**Discussion by Mawmewmaw** [marcussacana/MwareStuff#59] [Q&A]
uhm... im just trying to get text from Nitro+ game. I saw Mware stuff but didn't get how to use NUTEditor. I'm not really good at programming so i tried a few things which came on my mind but none of them worked. What should i do?

**Comment by marcussacana** [comment]
https://marcussacana.github.io/Sermone/

**Reply by Mawmewmaw** [reply]
Oh, tnank you so much! So, i get how this one works. I tried to open some of the files, it was fine for most of them, but in this one it doesn't show up the first phrase. I even checked it up with notepad, and yeah, can i somehow fix that?
![image](https://github.com/user-attachments/assets/ae00d925-528b-410c-85b2-5bd80ab3d4c2)



![image](https://github.com/user-attachments/assets/91515e64-c46f-4008-8c8e-005be6028b34)

**Reply by Mawmewmaw** [reply]
oh, seems like i somehow fixed that, everything is fine now

**Reply by hachim0on** [reply]
How did you fix it??? This is how everything is displayed for me. I don't understand programming either, if anything. But I need to work with the text too. The same picture is in Notepad++.
![image](https://github.com/user-attachments/assets/2c856ef3-4fa2-43a6-b65c-ea0480bb6da1)
